### PR TITLE
Add setup.py as well as empty README.md.

### PR DESCRIPTION
**Pull request by twiecki**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+AUTHOR = "Kevin Systrom, Thomas Vladeck"
+AUTHOR_EMAIL = "k@systrom.com"
+URL = "https://github.com/rtcovidlive/covid-model/"
+LICENSE = "Apache License, Version 2.0"
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
```
Comment (michaelosthege) [inline setup.py:32]: you can take out 3.6 ;)

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+#!/usr/bin/env python
+from codecs import open
+from os.path import realpath, dirname, join
+from setuptools import setup, find_packages
+import re
+
+DISTNAME = "covid"
```
Comment (michaelosthege) [inline setup.py:21]: the "covid" package name is already taken on PyPI. Maybe discuss a rename to "rtlive"?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        maintainer_email=AUTHOR_EMAIL,
+        description=DESCRIPTION,
+        license=LICENSE,
+        url=URL,
+        long_description=LONG_DESCRIPTION,
+        long_description_content_type="text/x-rst",
+        packages=find_packages(),
+        include_package_data=True,
+        classifiers=classifiers,
+        python_requires=">=3.6",
```
Comment (michaelosthege) [inline setup.py:81]: also here >=3.7  as we've already attempted to drop 3.6 for PyMC3

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    setup(
+        name=DISTNAME,
+        version=get_version(),
+        maintainer=AUTHOR,
+        maintainer_email=AUTHOR_EMAIL,
+        description=DESCRIPTION,
+        license=LICENSE,
+        url=URL,
+        long_description=LONG_DESCRIPTION,
+        long_description_content_type="text/x-rst",
```
Comment (michaelosthege) [inline setup.py:77]: Isn't the headling in README.md already in markdown?

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        packages=find_packages(),
+        include_package_data=True,
+        classifiers=classifiers,
+        python_requires=">=3.6",
+        install_requires=install_reqs,
+        tests_require=test_reqs,
+        test_suite="nose.collector",
```
Comment (canyon289) [inline setup.py:84]: Were not using nose

**Comment by twiecki** [comment]
Incorporated feedback.

**Comment by michaelosthege** [comment]
I'm merging this and will add more content to the README.md on another PR

**Pull request merged**